src/pages/AccountMigration.tsx

```diff
@@ -137,6 +142,8 @@ class AccountMigrationComponent extends React.Component<Props, State> {
       PhoneNumberFormat.E164
     );
 
+    this.formattedMobileNumber = formattedMobileNumber;
```

Comment:
Do we need the const anymore?

Reply:
mm, i guess not, but feels cleaner to me to just use the const instead of referring to this.formattedNumber everywhere.

Reply:
up to you though, i'm don't mind removing it either way

Reply:
nah sounds good to me!